cluster-autoscaler/expander/factory/expander_factory.go

```diff
@@ -48,12 +55,30 @@ func ExpanderStrategyFromString(expanderFlag string, cloudProvider cloudprovider
 			price.NewSimplePreferredNodeProvider(autoscalingKubeClients.AllNodeLister()),
 			price.SimpleNodeUnfitness), nil
 	case expander.PriorityBasedExpanderName:
-		maps := kubeClient.CoreV1().ConfigMaps(configNamespace)
-		initialPriorities, priorityChangesChan, err := priority.InitPriorityConfigMap(maps, configNamespace)
-		if err != nil {
-			return nil, errors.ToAutoscalerError(errors.InternalError, err)
-		}
-		return priority.NewStrategy(initialPriorities, priorityChangesChan, autoscalingKubeClients.LogRecorder)
+		// TODO: how to get proper termination info? It seems other listers do the same here
```

Comment:
Other listers seems to be doing the same - please make sure it's ok to leave the termination channel like that